app/lib/service/security_advisories/backend.dart

```diff
@@ -93,20 +96,65 @@ class SecurityAdvisoryBackend {
             osv.published != null ? DateTime.parse(osv.published!) : modified
         ..syncTime = syncTime;
 
-      tx.queueMutations(
-        // This is an upsert
-        inserts: [newAdvisory],
-      );
+      final packages = await _lookupAffectedPackages(newAdvisory, tx);
```

Comment:
What happens when there are too many affected packages?

Do we just never get a warning in the client?

Could we update the packages outside the transaction or rather in several smaller transactions?

Reply:
> Could we update the packages outside the transaction or rather in several smaller transactions?

Then it's not eventually consistent. We'd need more infrastructure to ensure evential consistency.

Let's hope we never get one that has more than 50 affected packages. And cross the bridge when that happens.